parser.py

```diff
@@ -0,0 +1,65 @@
+import sys
```

Comment:
Can we put this file into some reasonably name folder? Something like `utils`?
Also, maybe add some comment to the `main` method explaining what this actually does?

Reply:
I think the file is related to experiments.  We don't need to put it into master at all.

Reply:
Can we put it in `https://github.com/usi-verification-and-security/opensmt-debugging` ?
Seems like a useful thing to have available for the future.

Reply:
Sounds good to me!